lib/ui/window/platform_configuration_unittests.cc

```diff
@@ -332,5 +494,84 @@ TEST_F(PlatformConfigurationTest, SetDartPerformanceMode) {
   DestroyShell(std::move(shell), task_runners);
 }
 
+TEST_F(PlatformConfigurationTest, OutOfScopeRenderCallsAreIgnored) {
+  Settings settings = CreateSettingsForFixture();
+  TaskRunners task_runners = GetTaskRunnersForFixture();
+
+  MockRuntimeDelegate client;
+  auto platform_message_handler =
+      std::make_shared<MockPlatformMessageHandler>();
+  EXPECT_CALL(client, GetPlatformMessageHandler)
+      .WillOnce(Return(platform_message_handler));
+  // Render should not be called.
+  EXPECT_CALL(client, Render).Times(Exactly(0));
```

Comment:
Flutter engine code base doesn't appear to use `Exactly`: https://github.com/search?q=repo%3Aflutter%2Fengine+%22Exactly%28%22&type=code

```suggestion
  EXPECT_CALL(client, Render).Times(0);
```

Please update other locations too

Reply:
Sorry I missed this comment. I've removed it. Didn't know that Exactly is the default (I thought the default was AtLeast. :)